lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -1,38 +1,91 @@
+var assert = require('assert');
+
 module.exports = enforceExistence;
 module.exports.scopes = ['function'];
 module.exports.options = {
-    enforceExistence: {allowedValues: [true, 'exceptExports']}
+    enforceExistence: true
+};
+
+/**
+ * @param {Object} options
+ */
+enforceExistence.configure = function(options) {
+
+    // set default policy
+    enforceExistence.policy = {
```

Comment:
Please, use `this` here with `_` prefix for `policy` because there will be a conflict if somebody will use 2 different configurations at a time.